no function or associated item named `new_const` found for struct `std::fmt::Arguments<'a>` in the current scope

Hi,
with today's versions (rustc and creusot), I have an error in `creusot-std` when trying to prove a small project created by `cargo creusot new`:
```
$ cargo creusot new test_project && cd test_project && cargo creusot prove
[...]
   Compiling creusot-std-proc v0.9.0
    Checking creusot-std v0.9.0
error[E0599]: no function or associated item named `new_const` found for struct `std::fmt::Arguments<'a>` in the current scope
   --> /home/user/.cargo/registry/src/index.crates.io-1949cf8c6b5b557f/creusot-std-0.9.0/src/std/fmt.rs:142:20
    |
136 | / extern_spec! {
137 | |     mod core {
138 | |         mod fmt {
139 | |             impl<'a> Arguments<'a> {
140 | |                 #[check(ghost)]
141 | |                 #[requires(N <= 1usize)]
142 | |                 fn new_const<const N: usize>(pieces: &'a [&'static str; N]) -> Self;
    | |                   -^^^^^^^^^ function or associated item not found in `std::fmt::Arguments<'a>`
    | |___________________|
    |
    |
note: if you're trying to build a new `std::fmt::Arguments<'a>` consider using one of the following associated functions:
      std::fmt::Arguments::<'a>::new
      std::fmt::Arguments::<'a>::from_str_nonconst
      std::fmt::Arguments::<'a>::from_str
   --> /rustc/de6d33c033441c5880b863f94d7a3ec8cad141bd/library/core/src/fmt/mod.rs:729:4
```

Versions
```
$ cargo creusot version
cargo-creusot 0.9.0
Rust toolchain nightly-2026-01-29
Why3 platform, version 1.8.2+git
why3find v1.2.0+dev
alt-ergo 2.6.2
z3 4.15.3
cvc4 1.8
cvc5 1.3.1
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -17,7 +17,7 @@ members = [
 ]
 
 [workspace.package]
-version = "0.9.0"
+version = "0.10.0-dev"
 documentation = "https://creusot-rs.github.io/creusot/doc/creusot_std/"
 readme = "README.md"
 keywords = ["verification"]

--- a/cargo-creusot/src/main.rs
+++ b/cargo-creusot/src/main.rs
@@ -343,7 +343,7 @@ fn check_contracts_version() -> Result<()> {
         bail!(
             r"{msg}
     creusot-std {contracts_version}
-    creusot           {self_version}
+    creusot     {self_version}
 {fixes}"
         )
     };

--- a/creusot-args/Cargo.toml
+++ b/creusot-args/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "creusot-args"
-version = "0.9.0"
+version = "0.10.0-dev"
 edition = "2024"
 publish = false
 

--- a/creusot-rustc/Cargo.toml
+++ b/creusot-rustc/Cargo.toml
@@ -8,7 +8,7 @@ publish = false
 
 [dependencies]
 serde_json = { version = "1.0" }
-creusot = { path = "../creusot", version = "0.9.0" }
+creusot = { path = "../creusot", version = "0.10.0-dev" }
 env_logger = "0.11"
 serde = { version = "1.0", features = ["derive"] }
 creusot-args = { path = "../creusot-args" }

--- a/creusot-setup/Cargo.toml
+++ b/creusot-setup/Cargo.toml
@@ -1,6 +1,6 @@
 [package]
 name = "creusot-setup"
-version = "0.9.0"
+version = "0.10.0-dev"
 edition = "2024"
 publish = false
 

--- a/creusot-std-proc/Cargo.toml
+++ b/creusot-std-proc/Cargo.toml
@@ -27,7 +27,7 @@ creusot = ["dep:uuid", "dep:pearlite-syn", "proc-macro2/span-locations"]
 [dependencies]
 quote = "1.0"
 uuid = { version = "1.12", features = ["v4"], optional = true }
-pearlite-syn = { version = "0.9.0", path = "../pearlite-syn", features = ["full"], optional = true }
+pearlite-syn = { version = "0.10.0-dev", path = "../pearlite-syn", features = ["full"], optional = true }
 syn = { version = "2.0", features = ["full", "visit", "visit-mut"] }
 proc-macro2 = { version = "1.0" }
 

--- a/creusot-std/Cargo.toml
+++ b/creusot-std/Cargo.toml
@@ -18,7 +18,7 @@ categories.workspace = true
 num-rational = { version = "0.4", features = ["num-bigint"], default-features = false }
 
 [dependencies]
-creusot-std-proc = { path = "../creusot-std-proc", version = "0.9.0" }
+creusot-std-proc = { path = "../creusot-std-proc", version = "0.10.0-dev" }
 
 [features]
 # Enabled by creusot.

